packages/terra-hookshot/src/Hookshot.jsx

```diff
@@ -73,12 +73,20 @@ const propTypes = {
    */
   onPosition: PropTypes.func,
   /**
-   * Required element that the content will hookshot to.
+   * Client coordinates to serve as the anchor point for the hookshot'd content.
    */
-  targetRef: PropTypes.func.isRequired,
+  targetCoordinates: PropTypes.shape({
+    x: PropTypes.number,
+    y: PropTypes.number,
+  }),
+  /**
+   * Element to serve as the anchor point for the hookshot'd content. (If targetCoordinates are provided, this is ignored.)
+   */
+  targetRef: PropTypes.func,
   /**
    * Object containing the vertical and horizontal attachment values for the target.
    * Valid values: { horizontal: ['start', 'center', 'end'], vertical: ['top', 'middle', 'bottom'] }.
+   * If targetCoordinates are provided { horizontal: 'center', vertical: 'middle' } will be applied.
```

Comment:
provided by default or is this the only option when using target coordinates?

Reply:
Since it'll be positioning with a single coordinate, we don't want to worry about non-standard positioning on a single point. Under the hood it uses a 2x2px grid to position it with origin at 1x1, in order to avoid rounding errors.